### ISSUE TITLE
Android app

### ISSUE THREAD

**Issue by luke-jr** [bitcoinknots/bitcoin#67]
It would be good to have a usable Android app

**Comment by chrisguida** [comment]
Would this be a full node on the phone? Or would it merely connect to the node being run on a desktop?

**Comment by luke-jr** [comment]
Full node on the phone

**Comment by luke-jr** [comment]
Note that the codebase in theory already supports this

**Comment by vincenzopalazzo** [comment]
Well we should consider that these days Android is not a phone only OS 😄

**Comment by chrisguida** [comment]
@vincenzopalazzo interesting, what other devices run Android?

**Comment by vincenzopalazzo** [comment]
Chrome OS is running android app for example, I have a pixel book where I use green for android

**Comment by securitybrahh** [comment]
for mobiles, this aproach is better no? 

https://github.com/lightninglabs/neutrino

https://github.com/utreexo/utreexod

https://bitcoinops.org/en/topics/transaction-bloom-filtering/

**Comment by luke-jr** [comment]
No, Neutrino cannot be used for syncing a full node and is therefore not secure. And if you have your full node at home, the Bloom protocol is more optimal for a mobile app. But many people don't have home computers anymore, so Knots should support them natively.

**Comment by securitybrahh** [comment]
What about this?

https://github.com/vinteumorg/floresta

**Comment by kaisarrmeoydw** [comment]
look at this https://github.com/smartm0use/smartino
